src/AspNetCoreMulti/Example/Repositories/DogRepository.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Example.Repositories
```

Comment:
Please use file-scoped namespaces. We will convert codebase to use filescoped namespaces some time.

Reply:
I mean only for new files. Or just leave it as is, not a big deal.